api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/HawkularMetricsRestApp.java

```diff
@@ -110,7 +111,8 @@ public HawkularMetricsRestApp() {
         classes.add(EmptyPayloadFilter.class);
         classes.add(TenantFilter.class);
         classes.add(MetricsServiceStateFilter.class);
-        classes.add(CorsFilter.class);
```

Comment:
I am confused. I think you answered this in your last comment regarding the CORS implementation for JAX-RS 1.1:

@tsegismont  wrote:
>Oh, my bad, I did a mistake while looking at those classes in IDEA. Ok for servlet, there's no other way.